rskj-core/src/main/java/co/rsk/cli/tools/IndexBlooms.java

```diff
@@ -27,25 +28,29 @@
 import org.slf4j.LoggerFactory;
 
 import javax.annotation.Nonnull;
+import java.lang.invoke.MethodHandles;
 
 /**
  * The entry point for indexing block blooms
  * This is an experimental/unsupported tool
  */
-public class IndexBlooms {
+public class IndexBlooms extends CliToolRskContextAware {
 
     private static final Logger logger = LoggerFactory.getLogger(IndexBlooms.class);
 
     private static final String EARLIEST = "earliest";
     private static final String LATEST = "latest";
 
     public static void main(String[] args) {
-        try (RskContext ctx = new RskContext(args)) {
+        create(MethodHandles.lookup().lookupClass()).execute(args);
+    }
+
+    @Override
+    protected void onExecute(@Nonnull String[] args, @Nonnull RskContext ctx) throws Exception {
             BlockStore blockStore = ctx.getBlockStore();
             BlocksBloomStore blocksBloomStore = ctx.getBlocksBloomStore();
 
             execute(makeBlockRange(args, blockStore), blockStore, blocksBloomStore);
```

Comment:
could you please fix the indentation here?

Reply:
Done